competency-explorer/src/components/FilterBar/DisplayMode/styles.scss

```diff
@@ -0,0 +1,21 @@
+.displayModes {
+    margin: 1rem;
+    display: flex;
+    flex-direction: column;
+
+    .displayMode {
+        cursor: pointer;
+        width: auto;
+        border-radius: 5px;
+        padding: 0.1rem 0.25rem;
+
+        &:not(.selected) {
+            opacity: 0.5;
+        }
+
+        &.selected {
+            background-color: #FFF;
+            color: #333
```

Comment:
Bonus points, nonblocking IMO: it'd be a little more maintainable if these common constants such as colors and font families were moved to a shared constants file, similar to how we have it in the monolith (even bonus points for adding Gamut, but that seems fine to me as a secondary followup).

Reply:
agree & will add to the next PR